README.md

```diff
@@ -8,13 +8,18 @@ Caikit-NLP implements concept of "task" from `caikit` framework to define (and c
-| Text Classification  | 1. `SequenceClassification`               | 1. (Work in progress..)                                                                                                                                     |
-| Token Classification | 1. `FilteredSpanClassification`           | 1. (Work in progress..)                                                                                                                                     |
-| Tokenization         | 1. `RegexSentenceSplitter`                | 1. Demo purposes only                                                                                                                                       |
-| Embedding         | [COMING SOON]                | [COMING SOON]                                                                                                                                       |
+| Task                    | Module(s)                                      | Salient Feature(s)                                                                                                                                                         |
+|-------------------------|------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| TextGenerationTask      | 1. `PeftPromptTuning` <br> 2. `TextGeneration` | 1. Prompt Tuning, Multi-task Prompt tuning <br> 2. Fine-tuning  Both modules above provide optimized inference capability using Text Generation Inference Server           |
+| TextClassificationTask  | 1. `SequenceClassification`                    | 1. (Work in progress..)                                                                                                                                                    |
+| TokenClassificationTask | 1. `FilteredSpanClassification`                | 1. (Work in progress..)                                                                                                                                                    |
+| TokenizationTask        | 1. `RegexSentenceSplitter`                     | 1. Demo purposes only                                                                                                                                                      |
+| EmbeddingTask           | 1. `TextEmbedding`                             | 1. text/embedding from a local sentence-transformers model                                                                                                                 
+| EmbeddingTasks          | 1. `TextEmbedding`                             | 1. Same as EmbeddingTask but multiple sentences (texts) as input and corresponding list of outputs.                                                                        
```

Comment:
Can we combine these? Like we do for prompt tuning?

Reply:
updated README.md please take a look